client/my-sites/checkout/composite-checkout/lib/get-plan-features.ts

```diff
@@ -67,11 +68,10 @@ export default function getPlanFeatures(
 	if ( isWpComBusinessPlan( productSlug ) || isWpComProPlan( productSlug ) ) {
 		return [
 			! isMonthlyPlan && freeOneYearDomain,
-			! isMonthlyPlan && liveChatSupport,
-			String( translate( 'Install custom plugins and themes' ) ),
-			String( translate( 'Drive traffic to your site with our advanced SEO tools' ) ),
-			String( translate( 'Track your stats with Google Analytics' ) ),
-			String( translate( 'Real-time backups and activity logs' ) ),
+			String( translate( 'Install Custom Plugins and themes' ) ),
```

Comment:
Is it intentional to capitalize "Custom Plugins" this way? I would have expected "custom plugins", but defer to the copy editors.